src/ripple/protocol/impl/AccountID.cpp

```diff
@@ -25,9 +25,135 @@
 
 namespace ripple {
 
+#ifndef RIPPLED_ACCOUNTID_CACHE_SIZE
+#define RIPPLED_ACCOUNTID_CACHE_SIZE 1000000
+#endif
+
+namespace detail {
+
+/** Caches the base58 representations of AccountIDs */
+class AccountIdCache
+{
+private:
+    // The total number of items the cache buffer can contain
+    std::size_t const count_;
+
+    // The maximum size (in bytes) of an item, along with a null terminator
+    // and aligned at 8 bytes
+    std::size_t const size_ = 40;
```

Comment:
@scottschurr given I'll be closing this PR soon (and it might not get looked at again after that), would it be worth [opening an issue](https://github.com/XRPLF/rippled/issues/new/choose) for this?

If not, we can leave as-is.

Reply:
It's not worth an issue.  We can leave it as is.